library/src/test/java/com/getstream/sdk/chat/view/DefaultScrollButtonBehaviourTest.kt

```diff
@@ -0,0 +1,95 @@
+package com.getstream.sdk.chat.view
+
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import com.nhaarman.mockitokotlin2.doReturn
+import com.nhaarman.mockitokotlin2.times
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.whenever
+import org.junit.Test
+import org.mockito.Mockito
+
+private const val NEW_MESSAGE_SINGULAR: String = "%s New Message"
+private const val NEW_MESSAGES_PLURAL: String = "%s New Messages"
+
+internal class DefaultScrollButtonBehaviourTest {
+
+    private val unseenBottomBtn: ViewGroup = Mockito.mock(ViewGroup::class.java)
```

Comment:
Can be done like this using `mockito-kotlin` (just needs `import com.nhaarman.mockitokotlin2.mock`)

```suggestion
    private val unseenBottomBtn: ViewGroup = mock()
```